apps/lend/src/components/PageMarketList/index.tsx

```diff
@@ -40,17 +41,14 @@ const MarketList = (pageProps: PageMarketList) => {
   const TABLE_LABELS: { borrow: TableLabel[]; supply: TableLabel[] } = {
```

Comment:
looks like an object that gets recreated every render, that might cause performance problems. useMemo?